pkgs/top-level/all-packages.nix

```diff
@@ -4112,7 +4112,9 @@ with pkgs;
 
   jellyfin-mpv-shim = python3Packages.callPackage ../applications/video/jellyfin-mpv-shim { };
 
-  jellyfin-web = callPackage ../servers/jellyfin/web.nix { };
+  jellyfin-web = callPackage ../servers/jellyfin/web.nix {
+    buildNpmPackage = buildNpmPackage.override { nodejs = nodejs-14_x; };
```

Comment:
I should probably comment why this is done (Jellyfin's web client uses lockfile v1 at this tag, which doesn't work well on newer versions with our setup).

Maybe I can also add a hint for this somewhere, based on the version? Or maybe just in docs.

Reply:
Maybe it'd be better to have an attribute along the lines of `nodeJs` which defines the version used. 

That would make it more clear from the child derivation which version is used, and make changing it (upon a package update or something) more intuative.

Reply:
I'm confused -- what do you mean by this, where would that attribute be located? Can you provide an example of what that API would look like?

Reply:
Ah right, I forgot other builds do it like that. Creating `buildNpmPackage14` etc sounds like a good idea 👍